dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/SKChatMessage.cs

```diff
@@ -7,11 +7,11 @@
 namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.AzureSdk;
 
 /// <summary>
-/// Chat message representation from Semantic Kernel ChatMessageBase Abstraction
+/// Chat message representation from Semantic Kernel ChatMessage Abstraction
 /// </summary>
-public class SKChatMessage : ChatMessageBase
+public class SKChatMessage : SemanticKernel.AI.ChatCompletion.ChatMessage
 {
-    private readonly ChatMessage? _message;
+    private readonly Azure.AI.OpenAI.ChatMessage? _message;
```

Comment:
Would it make sense to replace the `public FunctionCall FunctionCall { get; }` property with `public Azure.AI.OpenAI.ChatMessage Message { get; }`? i.e. expose the whole message rather than just a piece of it, in order to provide any addditional further details a developer might want to see?

Reply:
We'd have to move the logic to throw the exception also.  I've created a task to follow up on this: https://github.com/microsoft/semantic-kernel/issues/3521